app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb

```diff
@@ -1,2 +1,83 @@
 class ManageIQ::Providers::Amazon::CloudManager::CloudDatabase < ::CloudDatabase
+  supports :create
+
+  def self.params_for_create
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :id         => 'name',
+          :name       => 'name',
+          :label      => _('Cloud Database Name'),
+          :isRequired => true
+        },
+        {
+          :component  => 'text-field',
+          :name       => 'storage',
+          :id         => 'storage',
+          :label      => _('Storage (in Gigabytes)'),
+          :type       => 'number',
+          :step       => 1,
+          :isRequired => true,
+          :validate   => [{:type => 'required'},
+                          {:type => 'min-number-value', :value => 1, :message => _('Size must be greater than or equal to 1')}],
+        },
```

Comment:
There are storage limitations depending on which database the user has selected, not sure how to enforce that or if we should https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/RDS/Client.html#create_db_instance-instance_method

Reply:
This is fine for now we don't have to try to replicate every business rule on the aws side (especially b/c they might change and we'd have to keep in sync)